### PR TITLE
yearsoo instead of year

### PR DESCRIPTION
**Pull request by Hussplus**

### CODE REVIEW

**Comment by Hussplus** [comment]
@jogermat what do you think

**Comment by Hussplus** [comment]
@jogermat you know what to do

File: banana.R [hunk 1]
```diff
@@ -1,3 +1,8 @@
 
 data <- le_data
 str(data)
+
+mean(data$year)
```
Comment (jogermat) [inline banana.R:5]: I think I can finally make review comments!

File: banana.R [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+
+data2 <- data %>% rename(yearsoo = year)
```
Comment (jogermat) [inline banana.R:7]: terrible variable name

Comment (Hussplus) [reply banana.R:7]: you name it next time :/

**Review by jogermat — APPROVED** [review]
done!

**Pull request merged**